src/plugins/telemetry_management_section/public/plugin.tsx

```diff
@@ -38,16 +38,32 @@ export interface TelemetryManagementSectionPluginDepsSetup {
   advancedSettings: AdvancedSettingsSetup;
 }
 
-export class TelemetryManagementSectionPlugin implements Plugin {
+export interface TelemetryManagementSectionPluginSetup {
+  toggleSecuritySolutionExample: (enabled: boolean) => void;
+}
+
+export class TelemetryManagementSectionPlugin
+  implements Plugin<TelemetryManagementSectionPluginSetup> {
+  private showSecuritySolutionExample = false;
+  private shouldShowSecuritySolutionExample = () => {
+    return this.showSecuritySolutionExample;
+  };
+
   public setup(
     core: CoreSetup,
     { advancedSettings, telemetry: { telemetryService } }: TelemetryManagementSectionPluginDepsSetup
   ) {
     advancedSettings.component.register(
       advancedSettings.component.componentType.PAGE_FOOTER_COMPONENT,
-      telemetryManagementSectionWrapper(telemetryService),
+      telemetryManagementSectionWrapper(telemetryService, this.shouldShowSecuritySolutionExample),
```

Comment:
I am wondering if you really need to pass a function here, I feel like you can just pass `this.showSecuritySolutionExample`

Reply:
Since booleans are copied and not passed by reference; we need a function to get the latest value of the variable. Passing it as a value directly might cause unexpected bugs especially since most kibana does not referesh between pages now.